sections/dom.include

```diff
@@ -604,140 +604,75 @@
   certain meanings (semantics). For example, the <{ol}> element represents an ordered list,
   and the <{global/lang}> attribute represents the language of the content.
 
-  These definitions allow HTML processors, such as Web browsers or search engines, to present and
-  use documents and applications in a wide variety of contexts that the author might not have
-  considered.
+  These definitions allow HTML processors, like browsers and search engines, to present documents and applications consistently in different contexts.
 
   <div class="example">
-    As a simple example, consider a Web page written by an author who only considered desktop
-    computer Web browsers:
+  <p>In this example the HTML headings may be presented as large text in a desktop browser, or standard size text in bold in a mobile browser. In both cases the semantic information remains the same - that the <{h1}> and <{h2}> elements represent headings.</p>
 
     <pre highlight="html">
-      &lt;!DOCTYPE HTML>
-      &lt;html>
-        &lt;head>
-          &lt;title>My Page&lt;/title>
-        &lt;/head>
-        &lt;body>
-          &lt;h1>Welcome to my page&lt;/h1>
-          &lt;p>I like cars and lorries and have a big Jeep!&lt;/p>
-          &lt;h2>Where I live&lt;/h2>
-          &lt;p>I live in a small hut on a mountain!&lt;/p>
-        &lt;/body>
-      &lt;/html>
+    &lt;!doctype html&gt;
+&lt;html lang="en"&gt;
```

Comment:
Indent this example so the root tags are 2 spaces in from the `<` of the `<pre>`. The remaining nesting looks correct.